bundles/org.openhab.binding.mikrotik/README.md

```diff
@@ -0,0 +1,374 @@
+
+## Supported Things
+
+* `routeros` - An instance of the RouterOS device connection
+* `interface` - A network interface inside RouterOS device
+* `wifiRegistration` - Any wireless client connected to a RouterOS wireless network (regular or CAPsMAN-managed)
+
+
+## Discovery
+
+Discovery is currently not supported, but may be implemented in future versions.
+
+
+## Binding Configuration
+
+The binding has no configuration options, all configuration is done at the Bridge and Thing levels.
+
+
+## Bridge Configuration
+
+To use this binding you need at least one RouterOS-powered device (Bridge) accessible to the host running
+OpenHAB via network.
+Make sure your RouterOS have API enabled by visiting [<kbd>IP -> Services</kbd>](https://wiki.mikrotik.com/wiki/Manual:IP/Services) 
+configuration section in 
+[WinBox](https://wiki.mikrotik.com/wiki/Manual:Winbox). Note the API port number as you'll need it below.
+[SSL API connection](https://wiki.mikrotik.com/wiki/Manual:API-SSL) is not yet supported by this binding.
+To connect to RouterOS API you need to provide user credantials to the bridge config. You may use your current 
+credentials you use to manage device, but it is highly recommended to **create a read-only RouterOS user** since
+this binding only reads data from the device. To do so, proceed to <kbd>System -> Users</kbd> configuration section
+and add a user to the `read` group.
+
+> Thing type: `routeros`
+
+The RouterOS Bridge configuration parameters are:
+
+* `host` _(required text)_ [Default: `192.168.88.1`] - Hostname or IP address of the RouterOS device
+* `port` _(optional integer)_ [Default: `8729`] - API Port number of the RouterOS device
+* `login` _(required text)_ [Default: `admin`] - The user login to access the the RouterOS device
+* `password` _(required text)_ - The user password to access the RouterOS device
+* `refresh` _(optional integer)_ [Default: `10`] - The refresh interval in seconds to poll the RouterOS device
+
+**All things provided by this binding requires a working bridge to be set up.**
+
+
+### Bridge Channels
+* <kbd>Number:DataAmount</kbd> `freeSpace` _(read-only)_ - Amount of free storage left on device in bytes
+* <kbd>Number:DataAmount</kbd> `totalSpace` _(read-only)_ - Amount of total storage available on device in bytes
+* <kbd>Number:Dimensionless</kbd> `usedSpace` _(read-only)_ - Percentage of used device storage space
+* <kbd>Number:DataAmount</kbd> `freeMemory` _(read-only)_ - Amount of free memory left on device in bytes
+* <kbd>Number:DataAmount</kbd> `totalMemory` _(read-only)_ - Amount of total memory available on device in bytes
+* <kbd>Number:Dimensionless</kbd> `usedMemory` _(read-only)_ - Percentage of used device memory
+* <kbd>Number:Dimensionless</kbd> `cpuLoad` _(read-only)_ - CPU load percentage
+* <kbd>String</kbd> `uptime` _(read-only)_ - For how long the device is working
+* <kbd>DateTime</kbd> `upSince` _(read-only)_ - Since when the device is working
```

Comment:
Are both of these needed? Also how often does the upTime change, is it every second?

Reply:
They are, as `uptime` is the raw _string_ representing the duration
![image](https://user-images.githubusercontent.com/81665/111929049-fe9baf80-8abd-11eb-9d07-9e51712915c9.png)

while `upSince` is the _DateTime_ value which is calculated as _now - uptime`
![image](https://user-images.githubusercontent.com/81665/111929137-30ad1180-8abe-11eb-8691-eaf7bd4dd761.png)

Value is refreshed each time the binding gets the new data (_Refresh interval_ config setting).